.changeset/grumpy-mirrors-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@razorpay/blade': major
```

Comment:
```suggestion
'@razorpay/blade': minor
```

Is there any breaking change in the PR? we can do minor release if not

Reply:
No breaking changes @saurabhdaware . Changed it to minor, please check.

Reply:
awesome. PR looks good to me. We'll discuss with the team how we want to go with icons with similar names and update here and in the thread.